package.json

```diff
@@ -27,5 +27,8 @@
     "postinstall": "yarn install:client && yarn install:server",
     "install:client": "cd client && yarn install",
     "install:server": "cd server && yarn install"
+  },
+  "dependencies": {
```

Comment:
I'm little skeptic about keeping dependencies in the root.
This will make all projects install root dependecy before deploying the actual code.

For devDependencies it seems okay, as we run them locally. Depending on deplyment strategy we can not be sure, that this dependency will be fetched for each app.

Reply:
Removing this dependency from the root folder.
Thanks for pointing out.